examples/apache_metrics.mtail

```diff
@@ -25,85 +28,90 @@ counter apache_http_sent_bytes_total by server_port, handler, request_method, re
+    http_request_duration_seconds_bucket["0.25"][$server_port][$handler][$method][$code][$protocol]++
   }
 
   # 500ms bucket.
   $time_us <= 500000 {
-    apache_http_request_time_microseconds["500000"][$server_port][$handler][$request_method][$request_status][$request_protocol]++
+    http_request_duration_seconds_bucket["0.5"][$server_port][$handler][$method][$code][$protocol]++
   }
 
   # 1s bucket.
   $time_us <= 1000000 {
-    apache_http_request_time_microseconds["1000000"][$server_port][$handler][$request_method][$request_status][$request_protocol]++
+    http_request_duration_seconds_bucket["1"][$server_port][$handler][$method][$code][$protocol]++
   }
 
   # 2.5s bucket.
   $time_us <= 2500000 {
-    apache_http_request_time_microseconds["2500000"][$server_port][$handler][$request_method][$request_status][$request_protocol]++
+    http_request_duration_seconds_bucket["2.5"][$server_port][$handler][$method][$code][$protocol]++
   }
 
   # 5s bucket.
   $time_us <= 5000000 {
-    apache_http_request_time_microseconds["5000000"][$server_port][$handler][$request_method][$request_status][$request_protocol]++
+    http_request_duration_seconds_bucket["5"][$server_port][$handler][$method][$code][$protocol]++
   }
 
   # 10s bucket.
   $time_us <= 10000000 {
-    apache_http_request_time_microseconds["10000000"][$server_port][$handler][$request_method][$request_status][$request_protocol]++
+    http_request_duration_seconds_bucket["10"][$server_port][$handler][$method][$code][$protocol]++
+  }
+
+  # 15s bucket.
+  $time_us <= 15000000 {
+    http_request_duration_seconds_bucket["15"][$server_port][$handler][$method][$code][$protocol]++
   }
 
   # "inf" bucket, also the total number of requests.
-  apache_http_request_time_microseconds["+Inf"][$server_port][$handler][$request_method][$request_status][$request_protocol]++
+  http_request_duration_seconds_bucket["+Inf"][$server_port][$handler][$method][$code][$protocol]++
 
   ###
   # Sent/Received bytes.
-  apache_http_sent_bytes_total[$server_port][$handler][$request_method][$request_status][$request_protocol] += $sent_bytes
-  apache_http_received_bytes_total[$server_port][$handler][$request_method][$request_status][$request_protocol] += $received_bytes
+  http_response_size_bytes_sum[$server_port][$handler][$method][$code][$protocol] += $sent_bytes
+  http_request_size_bytes_sum[$server_port][$handler][$method][$code][$protocol] += $received_bytes
 
   ### Connection status when response is completed:
   # X = Connection aborted before the response completed.
   # + = Connection may be kept alive after the response is sent.
   # - = Connection will be closed after the response is sent.
   / conn=X / {
-    apache_http_connections_total[$server_port][$handler][$request_method][$request_status][$request_protocol]["aborted"]++
+    http_connections_total[$server_port][$handler][$method][$code][$protocol]["aborted"]++
```

Comment:
I was thinking we should drop the label here, and just make it `http_connections_aborted_total`